### PR TITLE
PR: Add tests for Qtdesigner, QtNetwork, QtPrintSupport, QtSvg and QtTest.

### LINKED ISSUE #110: Add tests for untested modules
These modules are:

* QtDesigner
* QtNetwork
* QtOpenGL
* QtPrintSupport
* QtSvg
* QtTest

@rlaverde, you can look at `test_qtmultimedia` for inspiration. Please note that QtTest and QtDesigner are not available for PySide2.

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #110

I didn't add tests for QtOpenGL, because the usage of this module is discouraged http://doc.qt.io/qt-5/qtopengl-index.html

### CODE REVIEW

**Comment by goanpeca** [comment]
@rlaverde should we raise a warning to inform users of this??

File: qtpy/tests/test_qtdesigner.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import PYSIDE2, PYSIDE
+
+@pytest.mark.skipif(PYSIDE, reason="QtDesigner is not avalaible in PySide")
+@pytest.mark.skipif(PYSIDE2, reason="QtDesigner is not avalaible in PySide2")
```
Comment (goanpeca) [inline qtpy/tests/test_qtdesigner.py]: `@pytest.mark.skipif(PYSIDE2 or PYSIDE, reason="QtDesigner is not avalaible in PySide2")`

?

**Comment by rlaverde** [comment]
> should we raise a warning to inform users of this??

I'm not sure, maybe Qt has already a warning for this

**Comment by ccordoba12** [comment]
Please skip tests for QtSvg when using PySide2.

**Comment by goanpeca** [comment]
@ccordoba12, @rlaverde 

Should we create some tests that initialize objects?

like:

```python
from qtpy import QtWidget

def test_qtwidgets(qtbot):
    widget = QtWidget.QPushButton()
    assert widget
    widget = QtWidget.QComboBox()
    assert widget
    ...

```

Thoughts?

**Pull request merged**